libs/effects-ng/src/lib/tokens.ts

```diff
@@ -1,6 +1,9 @@
-import { InjectionToken } from '@angular/core';
+import { InjectionToken, Type } from '@angular/core';
+import { EffectsManager } from '@ngneat/effects';
 
-export const EFFECTS_PROVIDERS = new InjectionToken('EFFECTS_PROVIDERS');
-export const EFFECTS_MANAGER = new InjectionToken(
+export const EFFECTS_PROVIDERS = new InjectionToken<Type<any>[]>(
+  'EFFECTS_PROVIDERS'
+);
+export const EFFECTS_MANAGER = new InjectionToken<EffectsManager>(
   '@ngneat/effects Effects Manager'
```

Comment:
Let's be consistent with the EFFECTS_PROVIDERS name it EFFECTS_MANAGER instead of   '@ngneat/effects Effects Manager'